src/main/java/lotto/domain/WinningLotto.java

```diff
@@ -22,27 +22,26 @@ public WinningStatistics calculateStatistics(final List<Lotto> lottos) {
         }
 
         for (final Lotto lotto : lottos) {
-            Rank rank = checkRank(lotto.getNumbers(), winningNumbers.getWinningNumbers(), bonusNumber);
+            Rank rank = checkRank(lotto.getNumbers(), winningNumbers.getNumbers(), bonusNumber);
```

Comment:
2, 3 번째 메서드 파라미터는 어짜피 WinningLotto의 상태인데, 굳이 여기서 파라미터를 주입해줬을 필요가 잇나 싶어요. 🤔

또한 WinningLotto 안의 `checkRank`, `calculateMatchCount` 두 메서드는 불필요한 것 같아요. 
오히려 lotto 객체에게 calculateMatchCount의 메시지를 보냈다면 객체와 객체간의 협력이 조금 더 자연스러웠을 것 같아요.

```suggestion
            Rank rank = Rank.checkRank(lotto.calculateMatchCount(winningNumbers), lotto.contains(bonusNumber));
```

Reply:
로키가 말씀해주신 방향으로 고민해보고 수정했더니 WinningLotto와 Lotto 객체 간의 협력이 자연스러워졌어요.
거기다가 기존에 private 메서드였던 calculateMatchCount를 테스트할 수 있어져서 더욱 좋은 거 같아요!